packages/cubejs-server-core/src/core/CompilerApi.js

```diff
@@ -29,6 +30,11 @@ export class CompilerApi {
     this.sqlCache = options.sqlCache;
     this.standalone = options.standalone;
     this.nativeInstance = this.createNativeInstance();
+    this.compiledScriptCache = new LRUCache({
+      max: options.compilerCacheSize || 250,
+      ttl: options.maxCompilerCacheKeepAlive,
```

Comment:
AFAIR: We don't set up `maxCompilerCacheKeepAlive` by default. Could you check this moment? @KSDaemon

Reply:
And that is okay. It is optional. I just added it as we do in other places (for the similarity).